pg_variables.c

```diff
@@ -945,7 +949,27 @@ remove_package(PG_FUNCTION_ARGS)
 static void
 removePackageInternal(Package *package)
 {
-	TransObject *transObject;
+	TransObject			*transObject;
+	Variable			*variable;
+	HTAB				*htab;
+	HASH_SEQ_STATUS		 vstat;
+	int					 i;
+
+	/* Set all the variables from package is deleted */
```

Comment:
'all the variables ... is deleted' does not sound as a correct phrase for me. Maybe 'Mark all the variables from package as deleted'?

Reply:
Yeah, we set an appropriate flag to true, which mean variable is deleted.